packages/contracts-bedrock/src/dispute/interfaces/IFaultDisputeGame.sol

```diff
@@ -62,9 +62,6 @@ interface IFaultDisputeGame is IDisputeGame {
     /// @param _claimIndex The index of the subgame root claim to resolve.
     function resolveClaim(uint256 _claimIndex) external payable;
 
-    /// @notice A block hash on the L1 that contains the disputed output root.
```

Comment:
nit
```suggestion
    /// @notice A block hash on L1 that contains the disputed output root.
```

Reply:
Was changed in the `IDisputeGame` interface 👍